src/cdk/tree/tree.ts

```diff
@@ -333,8 +331,11 @@ export class CdkTreeNode<T> implements FocusableOption, OnDestroy {
   }
 
   get level(): number {
-    return this._tree.treeControl.getLevel ? this._tree.treeControl.getLevel(this._data) : 0;
-  }
+   // Retrieve the aria-level of the parent node because level from treeControl is 0 indexed and
```

Comment:
I think this comment should be clarified, maybe something like this:

"If the treeControl has a getLevel method, use it to get the level. Otherwise read the aria-level off the parent node and use it as the level for this node (note aria-level is 1-indexed, while this property is 0-indexed, so we don't need to incremenet)."